schemas/HPXMLBaseElements.xsd

```diff
@@ -5245,6 +5246,24 @@
 					<xs:attribute name="dataSource" type="DataSource"/>
 				</xs:complexType>
 			</xs:element>
+			<xs:element name="BellyAndWing">
+				<xs:annotation>
+					<xs:documentation>The floor is supported on a pair of I-beams spanning across the length of the manufactured home. The floor wing is the part of the floor outside the supports, and the floor belly is the part between the supports. The floor wing and belly sections are protected from outside elements including water, wind, and rodents using a wrap attached to the underside of floor joists, and may contain insulation. Use separate Floor elements to describe the floor insulation above the belly and wing sections respsectively.
```

Comment:
"Use separate Floor elements to describe the floor insulation above the belly and wing sections respsectively." -- That no longer reflects the current status of this PR, right?

Reply:
Removed.